### ISSUE TITLE
Possible XSS exploit

### ISSUE THREAD

**Issue by IvanRF** [stcr/subscribe-to-comments-reloaded#138]
I'm not an expert on this, but I saw you added `esc_attr()` on POST and GET. Shouldn't you use that also in [/options/index.pnp lines 70-71](https://github.com/stcr/subscribe-to-comments-reloaded/blob/pre_release/subscribe-to-comments-reloaded/options/index.php#L70)?

**Comment by Reedyseth** [comment]
> I'm not an expert on this, but I saw you added esc_attr() on POST and GET. Shouldn't you use that also in /options/index.pnp lines 68-71?

You are right. Any information coming from any source should be escape, but annotated.

**Comment by IvanRF** [comment]
Here are some more possible exploits: [/options/panel1-business-logic.php lines 94-98](https://github.com/stcr/subscribe-to-comments-reloaded/blob/pre_release/subscribe-to-comments-reloaded/options/panel1-business-logic.php#L94)

**Comment by IvanRF** [comment]
Also in

- [/options/panel1-edit-subscription.php lines 14 and 17](https://github.com/stcr/subscribe-to-comments-reloaded/blob/pre_release/subscribe-to-comments-reloaded/options/panel1-edit-subscription.php#L14)
- [/options/panel1.php line 7](https://github.com/stcr/subscribe-to-comments-reloaded/blob/pre_release/subscribe-to-comments-reloaded/options/panel1.php#L7)
- templates/author.php
- templates/one-click-unsubscribe.php
- templates/user.php
- wp_subscribe_reloaded.php lines 379-381 and 1044

Easiest way to find them is to search "$_GET" or "$_POST" in the whole project.
